pkg/apiserver/apiserver.go

```diff
@@ -324,6 +324,7 @@ func (app *cdiAPIApp) uploadHandler(request *restful.Request, response *restful.
 
 	tokenData := &token.Payload{
 		Operation: token.OperationUpload,
+		// this is service or pod but not pvc?
```

Comment:
No it is a PVC

Reply:
right it is then used to validate, comment removed